pyproject.toml

```diff
@@ -3,6 +3,156 @@
 requires = ['setuptools', 'wheel', 'plux>=1.7']
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "localstack-core"
+authors = [
+        { name = "LocalStack Contributors", email = "info@localstack.cloud" }
+]
+description = "The core library and runtime of LocalStack"
+requires-python = ">=3.8"
+dependencies = [
+    "build",
+    "click>=7.1",
+    "cachetools>=5.0",
+    "cryptography",
+    "dill==0.3.6",
+    "dnslib>=0.9.10",
+    "dnspython>=1.16.0",
+    "plux>=1.7",
+    "psutil>=5.4.8",
+    "python-dotenv>=0.19.1",
+    "pyyaml>=5.1",
+    "rich>=12.3.0",
+    "requests>=2.20.0",
+    "semver>=2.10",
+    "stevedore>=3.4.0",
+    "tailer>=0.4.1",
+]
+dynamic = ["version"]
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.11",
+    "License :: OSI Approved :: Apache Software License",
+    "Topic :: Internet",
+    "Topic :: Software Development :: Testing",
+    "Topic :: System :: Emulators",
+]
+
+[project.urls]
+Homepage = "https://localstack.cloud"
+Documentation = "https://docs.localstack.cloud"
+Repository = "https://github.com/localstack/localstack.git"
+Issues = "https://github.com/localstack/localstack/issues"
+
+[project.optional-dependencies]
+# minimal required to actually run localstack on the host for services natively implemented in python
+base-runtime = [
+    "awscrt>=0.13.14",
+    "boto3>=1.26.121",
+    "botocore==1.34.54",
+    "cbor2>=5.2.0",
+    "dnspython>=1.16.0",
+    # TODO tag incompatibility introduced in 7.0.0 with https://github.com/docker/docker-py/pull/3191
```

Comment:
it seems `localstack-twisted` vanished from `setup.cfg` and `pyproject.toml`? 

https://github.com/localstack/localstack/blob/0b9c6b1de4be4c8d24f52d6ba863f7085e0d0741/setup.cfg#L71

Reply:
Should be resolved now after rebasing